src/NHibernate/Linq/Visitors/SelectClauseNominator.cs

```diff
@@ -121,11 +121,11 @@ private bool IsRegisteredFunction(Expression expression)
 						   methodCallExpression.Object.NodeType != ExpressionType.Constant; // does not belong to parameter 
 				}
 			}
-			else if (expression.NodeType == (ExpressionType)NhExpressionType.Sum ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Count ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Average ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Max ||
-						expression.NodeType == (ExpressionType)NhExpressionType.Min)
+			else if (expression is NhSumExpression ||
+			         expression is NhCountExpression ||
+			         expression is NhAverageExpression ||
+			         expression is NhMaxExpression ||
+			         expression is NhMinExpression)
```

Comment:
Cast should be as cheap as the equal. 

The alignment were discussed somewhere with @oskarb. The consensus was that we use tabs for indents and spaces (after tabs) to align. This way it looks the same everywhere despite the size of a tab.